Source/WebCore/rendering/StyledMarkedText.cpp

```diff
@@ -52,7 +52,7 @@ static StyledMarkedText resolveStyleForMarkedText(const MarkedText& markedText,
     case MarkedText::Type::Highlight:
         if (auto renderStyle = renderer.parent()->getUncachedPseudoStyle({ PseudoId::Highlight, markedText.highlightName }, &renderer.style())) {
             style.backgroundColor = renderStyle->colorResolvingCurrentColor(renderStyle->backgroundColor());
-            style.textStyles.fillColor = renderStyle->computedStrokeColor();
+            style.textStyles.fillColor = renderStyle->color();
```

Comment:
could you clarify why is this being changed?

Reply:
Not sure if it should even be in this patch or different. `computedStrokeColor` shouldn't be setting fillColor.

Reply:
I think this should be in a separate patch, with a new or newly passing test.

Reply:
Yup I agree

Reply:
got it!